server/api/impl/store/store.go

```diff
@@ -13,6 +14,8 @@ const prefixSubs = "sub_"
 type Store struct {
 	mm   *pluginapi.Client
 	conf api.Configurator
+	// Manifests of the registered apps for this installation
+	manifests map[apps.AppID]*apps.Manifest
```

Comment:
A little confused here. is `manifests` a cache? Or is `Store` an in-memory only "store"? If it's the latter, aka not persistent, then perhaps it can just be a map or a struct? 1/5 `Store` indicates persistent storage.

Reply:
It's an in-memory store. This implementation is changed now. But I see store as an abstractions - no one should care weather it is an in-memory, persistent, file, db,...